recipes/CImg/build.bat

```diff
@@ -0,0 +1,6 @@
+@echo off 
+
+mkdir %PREFIX%/include/ 
```

Comment:
Is this the right place for headers on Windows?

* If it is, this file can be removed as `noarch: generic` packages are only build on Linux
* If it isn't, the package can't be `noarch: generic`

Reply:
I can only assume it is correct, I simply copied the build instructions from <https://anaconda.org/dtk-forge/cimg>. The guy who made this recipe also created the package. I will remove the `noarch` lines in meta.yaml